docs/general/occ.rst

```diff
@@ -26,31 +26,33 @@ most up to date cluster configuration.
 
 .. Hidden: update some documents to raise their ``_seq_no`` values.::
 
-    cr> update locations set date = 0
-    ... where name < 'Altair' and kind = 'Star System';
-    UPDATE OK, 3 rows affected (... sec)
+    cr> CREATE TABLE sensors (
+    ...   id text primary key,
+    ...   type text,
+    ...   last_verification timestamp
+    ... );
+    CREATE OK, 1 row affected  (... sec)
 
-    cr> update locations set date = 2
-    ... where name < 'Altair' and kind = 'Star System';
-    UPDATE OK, 3 rows affected (... sec)
+    cr> INSERT INTO sensors (id, type, last_verification) VALUES ('01DYHPT2Q36YQ5B72HC1H40KAA', 'DHT11', null);
```

Comment:
Can we use human-readable `id` values here instead of random chars? This would make the followup documentation more readable as it is more easy to see which document should be updated/deleted.